DocumentedExamples/Zonally_Averaged_Global_Meridional_Overturning_Circulation.ipynb

```diff
@@ -6,9 +6,9 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    psi_avg['potrho'] = (psi_avg.potrho-1028)**4
</pre><p>Could also save this as a new coordinate axis, and then switch that in as the coordinate axis for plotting, keeping the original intact</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>